docker-compose.yml

```diff
@@ -10,6 +10,8 @@ services:
       POSTGRES_PASSWORD: secret
     volumes:
     - hostedgpt_pgdata:/var/lib/postgresql/data
+    ports:
+    - 5433:5432
```

Comment:
What does this do? The docker compose has been working for me without this.

Reply:
not necessary, but useful if you want to connect to the postgres instance that's running in docker from your local machine via tools like postico. Happy to remove it if you want. I was just using it to look at my database

Reply:
Ahh, that makes sense. I think I used to have a couple ports exposed and then someone submitted a PR to remove them saying: it’s exposing ports which are conflicting with my local services and those ports arent being used.

So i think it helps ppl for us to expose less